rsocket-core/src/main/java/io/rsocket/RSocketFactory.java

```diff
@@ -104,11 +105,14 @@ public static ServerRSocketFactory receive() {
 
     private boolean resumeEnabled;
     private Supplier<ResumeToken> resumeTokenSupplier = ResumeToken::generate;
-    private int resumeCacheSize = 32768;
+    private Function<? super ResumeToken, ? extends ResumeStore> resumeStoreFactory =
+        token -> new InMemoryResumeStore("client", 32768);
```

Comment:
This seems like a really generous default, given each message could be quite large.  at 1k, this is 32mb per connection.

Reply:
At some point the pain of resumption may be worse than establishing a new connection for clients, e.g. if too much history is retained.

Reply:
I can change It to 1024?